src/ui/ContextSwitcher/ContextSwitcher.tsx

```diff
@@ -136,10 +174,8 @@ function ContextSwitcher({
         type="button"
         className={cs(
           '"relative flex gap-1 items-center text-xl font-semibold w-full rounded-md py-1.5 text-left text-gray-900 focus:outline-none"',
-          {
-            [buttonVariant.outlined]:
-              'ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-indigo-600 shadow-sm',
-          }
```

Comment:
this seemed like a bug... with `cs`, if we pass an obj, it applies the key as the class string if the value is truthy. Which means this was always just adding the classname "undefined"